src/lib.rs

```diff
@@ -14,8 +14,10 @@ pub use crate::{
     browser::dom::cast::{
         to_html_el, to_input, to_kbevent, to_mouse_event, to_select, to_textarea,
     },
+    browser::service::fetch,
     browser::service::fetch::{Method, Request, ResponseDataResult, ResponseResult},
```

Comment:
`browser::service::fetch::{self, Method, Request, ResponseDataResult, ResponseResult}` could be cleaner.